ansible/roles/test/files/ptftests/vnet_vxlan.py

```diff
@@ -307,7 +310,9 @@ def FromVM(self, test):
                 ip_dst=test['src'],
                 ip_src=test['dst'],
                 ip_id=108,
```

Comment:
too many parameters are set with hard coded values and repeated in different places of the test code.
suggest that all will be defined in one place and used by the different functions/files.